[build test failure] testDisclaimMemoryEffects_GC_Test can't shrink heap within 5 minutes

Test is failing in JDK9 builds on linux ppcle 64:
https://ci.eclipse.org/openj9/job/Test-Extended-JDK9-linux_ppc-64_cmprssptrs_le/94/console

```
02:46:57         [SoftmxAdvanceTest_GC_Only] [WARN] 	Generate Java core dump after forcing GC.
02:46:57         JVMJ9VM079W triggerOneOffDump(java) requires j9dmp29
02:46:57         JVMJ9VM079W triggerOneOffDump(heap) requires j9dmp29
02:46:57         JVMJ9VM079W triggerOneOffDump(system) requires j9dmp29
02:46:57         FAILED: testDisclaimMemoryEffects_GC_Test
02:46:57         java.lang.AssertionError: 	FAIL: Heap can't shrink to the reset max heap size within 5 minutes!
02:46:57         	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
02:46:57         	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
02:46:57         	at j9vm.test.softmx.SoftmxAdvanceTest_GC_Only.testDisclaimMemoryEffects_GC_Test(SoftmxAdvanceTest_GC_Only.java:150)
02:46:57         	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(java.base@9-internal/Native Method)
02:46:57         	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(java.base@9-internal/NativeMethodAccessorImpl.java:62)
02:46:57         	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(java.base@9-internal/DelegatingMethodAccessorImpl.java:43)
02:46:57         	at java.lang.reflect.Method.invoke(java.base@9-internal/Method.java:564)
02:46:57         	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
02:46:57         	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
02:46:57         	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
02:46:57         	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
02:46:57         	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
02:46:57         	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
02:46:57         	at org.testng.TestRunner.privateRun(TestRunner.java:648)
02:46:57         	at org.testng.TestRunner.run(TestRunner.java:505)
02:46:57         	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
02:46:57         	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
02:46:57         	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
02:46:57         	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
02:46:57         	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
02:46:57         	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
02:46:57         	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
02:46:57         	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
02:46:57         	at org.testng.TestNG.runSuites(TestNG.java:1049)
02:46:57         	at org.testng.TestNG.run(TestNG.java:1017)
02:46:57         	at org.testng.TestNG.privateMain(TestNG.java:1354)
02:46:57         	at org.testng.TestNG.main(TestNG.java:1323)
02:46:57         
02:46:57         
02:46:57         ===============================================
02:46:57             testSoftMxDisclaimMemory_GC
02:46:57             Tests run: 1, Failures: 1, Skips: 0
02:46:57         ===============================================
```

--- a/test/functional/JLM_Tests/src/j9vm/test/softmx/SoftmxAdvanceTest_GC_Only.java
+++ b/test/functional/JLM_Tests/src/j9vm/test/softmx/SoftmxAdvanceTest_GC_Only.java
@@ -31,7 +31,6 @@ import org.testng.AssertJUnit;
 import j9vm.test.softmx.MemoryExhauster;
 
 import java.lang.management.ManagementFactory;
-import java.lang.management.RuntimeMXBean;
 import java.util.Iterator;
 import java.util.List;
 
@@ -84,49 +83,16 @@ public class SoftmxAdvanceTest_GC_Only{
 		logger.debug("	Forcing Aggressive GC. Will wait a maximum of 5 minutes for heap shrink..." );
 		TestNatives.setAggressiveGCPolicy();
 
-		/* figure out DisclaimVirtualMemory setting from VM arguments.
-		 * The arguments will contain -Xsoftmx and -XX:+DisclaimVirtualMemory/-XX:-DisclaimVirtualMemory
-		 */
-		boolean enableDisclaimMemory = false;
-		boolean disableDisclaimMemory = false;
-		boolean softmxExist = false;
-
-		RuntimeMXBean RuntimemxBean = ManagementFactory.getRuntimeMXBean();
-		List<String> arguments = RuntimemxBean.getInputArguments();
-
-		Iterator it = arguments.iterator();
-
-		while (it.hasNext()){
-			String s = (String)it.next();
-			if (s.equalsIgnoreCase(DISCLAIM_MEMORY)){
-				enableDisclaimMemory = true;
-			}
-			if (s.equalsIgnoreCase(NOT_DISCLAIM_MEMORY)){
-				disableDisclaimMemory = true;
-			}
-			if (s.contains(SOFTMX)){
-				softmxExist = true;
-			}
-		}
-
 		long startTime = System.currentTimeMillis();
 		boolean isShrink = false;
 
 		//waiting for maximum 5 min (300000ms)
 		while((System.currentTimeMillis() - startTime) < 300000 ){
-			if ( ibmMemoryMBean.getHeapMemoryUsage().getCommitted() <= new_softmx_value ) {
-				if ( enableDisclaimMemory ) {
-					if ( ibmOSMBean.getFreePhysicalMemorySize() > preMemSize ) {
-						isShrink = true;
-					}
-				} else {
-					isShrink = true;
-				}
-				if ( isShrink == true ) {
-					logger.debug( "	PASS: Heap has shrunk to " + ibmMemoryMBean.getHeapMemoryUsage().getCommitted() + " bytes"
-					+ " in " + (System.currentTimeMillis() - startTime) + " mSeconds");
-					break;
-				}
+			if (ibmMemoryMBean.getHeapMemoryUsage().getCommitted() <= new_softmx_value) {
+				isShrink = true;
+				logger.debug( "	PASS: Heap has shrunk to " + ibmMemoryMBean.getHeapMemoryUsage().getCommitted() + " bytes"
+				+ " in " + (System.currentTimeMillis() - startTime) + " mSeconds");
+				break;
 			} else {
 				try {
 					Thread.sleep(2000);

